docs/stage2/tailscale.md

```
@@ -0,0 +1,152 @@
+
+```bash
+bws secret create TF_VAR_tailscale_auth_key "tskey-auth-..." "$BWS_PROJECT_ID"
+```
+
+**6. Apply stage 2**, then approve the advertised routes and the exit node offer for the new device on the [Machines](https://console.tailscale.com/admin/machines) page. An advertised route is inert until approved.
+
+## Gotchas
+
+!!! danger "The pod logs in again on every restart, so the key is not a one-shot bootstrap credential"
+
+    [`containerboot`](https://github.com/tailscale/tailscale/blob/v1.92.5/cmd/containerboot/main.go) documents `TS_AUTH_ONCE` as "if true, only attempt to log in if not already logged in. If false, the default, for backwards compatibility, forcibly log in every time the container starts." This module does not set it, and `tailscaleUp` appends `--authkey=` whenever the key is non-empty.
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)